macroslib/tests/test_expectations.rs

```diff
@@ -230,6 +230,25 @@ foreigner_class!(class TestEnumClass {
     }
 }
 
+#[test]
```

Comment:
I prefer usage of 
swig/macroslib/tests/expectations:

This need just creation of:

swig/macroslib/tests/expectations/fenum_with_comments.rs <- input
swig/macroslib/tests/expectations/fenum_with_comments.cpp <- output
and adding fenum_with_comments into
swig/macroslib/tests/expectations/tests.list